src/goto-programs/abstract-interpretation/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 add_library(abstract-interpretation ai.cpp ai_domain.cpp interval_domain.cpp interval_analysis.cpp)
 target_link_libraries(abstract-interpretation fmt::fmt)
 target_include_directories(abstract-interpretation
-        PUBLIC ${Boost_INCLUDE_DIRS}
+        PUBLIC ${Boost_INCLUDE_DIRS} ${IBEX_INCLUDE_DIRS} ${IBEX_INCLUDE_DIRS}/ibex ${IBEX_INCLUDE_DIRS}/ibex/3rd
```

Comment:
These should only be added if IBEX was found & enabled during the build.

Reply:
fixed.